src/components/LandingPage/FAQ/FAQ.tsx

```diff
@@ -12,18 +12,33 @@ interface FAQDropDownProps {
 
 const FAQDropDown: React.FC<FAQDropDownProps> = (props) => {
   const { question, answer } = props
+  let [opened, setOpen] = useState(false)
```

Comment:
Also, use ```const``` for useState() since  const prevents reassignment and we don't want to manipulate state directly by reassigning it. We want to call the setter function to update state instead.